.github/workflows/pr.yaml

```diff
@@ -128,31 +130,69 @@ jobs:
           node-version: 18
 
       - name: Install Playwright Browsers
-        run: npx playwright install --with-deps
+        run: npx playwright install --with-deps ${{ matrix.browser }}
 
       - name: Run Playwright tests
-        run: npx playwright test
-
-      - name: Save additional test information
-        if: always()
-        run: |
-          touch playwright-report/info.txt
-          echo "HTML_REPORT_URL_PATH:$HTML_REPORT_URL_PATH" >> playwright-report/info.txt
-          cat playwright-report/info.txt
+        run: npx playwright test --project ${{ matrix.browser }}
 
-      - uses: actions/upload-artifact@v4
+      - name: Upload ${{ matrix.browser }} blob reports to be merged
         if: always()
+        uses: actions/upload-artifact@v4
         with:
-          name: playwright-report
-          path: playwright-report/
-          retention-days: 30
+          name: blob-report-${{ matrix.browser }}
+          path: blob-report
+          retention-days: 1
```

Comment:
This is for retaining the blob data which is then used to merge into the final html report. We don't really want to keep this data for long since it serves no purpose after it is merged. The final reports are retained for 14 days (2 weeks) which I think is reasonable, previously the retention period of 30 days seems really long so I adjusted it.

Reply:
I see, thanks for the explanation. Makes sense :+1: